D1673/P1673.md

```diff
@@ -8882,49 +8865,37 @@ void hermitian_matrix_rank_1_update(
   Triangle t);
 ```
 
-<i>[Note:</i> These functions correspond to the BLAS functions
-`xHER` and `xHPR`.
+[12]{.pnum} These functions perform a Rank-1 update of a Hermitian matrix taking into account the `Triangle` parameter [linalg.general].
 
-They have overloads taking a scaling factor `alpha`,
-because it would be impossible to express the update
-$A = A - x x^H$ otherwise. <i>-- end note]</i>
+[13]{.pnum} *Effects:* Computes $A'$ such that
+$A' = A + \alpha x x^H$, where the scalar $\alpha$ is `alpha`, and assigns each element of $A'$ to the corresponding element of $A$.
 
-[1]{.pnum} *Constraints:*
-If `A` has `layout_blas_packed` layout,
-then the layout's `Triangle` template argument
-has the same type as
-the function's `Triangle` template argument.
+#### Rank-2 matrix updates [linalg.algs.blas2.rank2]
 
-[2]{.pnum} *Mandates:*
+<i>[Note:</i> These functions correspond to the BLAS functions
```

Comment:
Do Notes need to belong to a numbered paragraph?  I'm asking because this one precedes Paragraph 1.